haas/cli.py

```diff
@@ -66,7 +67,11 @@ def check_status_code(response):
 # TODO: This function's name is no longer very accurate.  As soon as it is
 # safe, we should change it to something more generic.
 def object_url(*args):
-    url = cfg.get('client', 'endpoint')
+    # Prefer an environmental variable for getting the endpoint if available.
+    url = os.environ.get('HAAS_ENDPOINT')
+    if url == None:
```

Comment:
[pep8 says][1] you should use `is` here:

`if url is None`

[1]: https://www.python.org/dev/peps/pep-0008/#id42